### PR TITLE
Add Execute Command to API documentation.

### LINKED ISSUE #614: Add Execute API to ML Commons
## What do you want to do?
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request. For example, what feature or area does it affect? What versions? If it’s a change to existing documentation, which pages (URLs) are affected?**

URL: https://opensearch.org/docs/latest/ml-commons-plugin/api/

The ML Commons team needs the Execute API documented for v2.0.0. We also need to add a note to Delete API stating "We don't check task status when delete task. So you can delete a running task but you will not be able to get the task status to know if it's completed or not. So be careful when delete task".

**Do you have any related resources to help us get started (e.g., link to a related fixed issue, design doc, etc.)? Any particular POCs (Eng or PM) who have more information?**

POC: @ylwu-amzn

Also see the comment on the Supported Algorithms PR for an execute API example: https://github.com/opensearch-project/documentation-website/pull/597#discussion_r882967372

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Signed-off-by: Naarcha-AWS <naarcha@amazon.com>

Closes #614 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Naarcha-AWS** [comment]
@ylwu-amzn: This is ready for your review.

**Comment by Naarcha-AWS** [comment]
@natebower: This is ready for your review.

File: _ml-commons-plugin/algorithms.md [hunk 1]
```diff
@@ -198,9 +198,9 @@ time_zone | string | The time zone for the time_field field | "UTC"
 
 For FIT RCF, you can train the model with historical data and store the trained model in your index. The model will be deserialized and predict new data points when using the Predict API. However, the model in the index will not be refreshed with new data, because the model is fixed in time.
 
-## Anomaly Localization 
+## Localization 
 
-The Anomaly Localization algorithm finds subset level-information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies. Localization can be applied in different scenarios, such as data exploration or root cause analysis, to expose the contributors driving the activity of interest in the aggregate data.
+The Localization algorithm finds subset level-information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies. Localization can be applied in different scenarios, such as data exploration or root cause analysis, to expose the contributors driving the activity of interest in the aggregate data.
```
Comment (natebower) [inline _ml-commons-plugin/algorithms.md:203]: It looks like this should be "subset-level information" (move hyphen).

File: _ml-commons-plugin/api.md [hunk 2]
```diff
@@ -644,6 +644,9 @@ GET /_plugins/_ml/tasks/_search
 
 Delete a task based on the task_id.
 
+{: .note}
+ML commons does not check the task status when running the `Delete` request. There is a risk that a currently running task could be deleted before the task completes. To check the status of a task, run `GET /_plugins/_ml/tasks/<task_id>` before task deletion.
```
Comment (natebower) [inline _ml-commons-plugin/api.md:648]: Capitalize "Commons".

File: _ml-commons-plugin/api.md [hunk 3]
```diff
@@ -729,8 +732,147 @@ GET /_plugins/_ml/stats
...
 
+## Execute
+
+Some algorithms, such as [Localization]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms#localization), don't require trained models. You can run no-model-based algorithms using the `execute` API.
 
+```json
+POST _plugins/_ml/_execute/<algorithm_name>
+```
 
+### Example: Execute Localization
```
Comment (natebower) [inline _ml-commons-plugin/api.md:743]: Sentence case

File: _ml-commons-plugin/api.md [hunk 4]
```diff
@@ -729,8 +732,147 @@ GET /_plugins/_ml/stats
...
+
+Some algorithms, such as [Localization]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms#localization), don't require trained models. You can run no-model-based algorithms using the `execute` API.
 
+```json
+POST _plugins/_ml/_execute/<algorithm_name>
+```
 
+### Example: Execute Localization 
 
+The following example uses localization to find subset level information for aggregate data (for example, aggregated over time) that demonstrates activity of interest such as spikes, drops, changes, and anomalies.
```
Comment (natebower) [inline _ml-commons-plugin/api.md:745]: Change to match the earlier wording: "The following example uses the Localization algorithm to find subset-level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies."

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS Please see my comments and let me know if you have any questions. Thanks!

**Review by natebower — APPROVED** [review]
LGTM, assuming the two changes we discussed are made.

File: _ml-commons-plugin/api.md [hunk 5]
```diff
@@ -644,6 +644,9 @@ GET /_plugins/_ml/tasks/_search
 
 Delete a task based on the task_id.
 
+{: .note}
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/api.md:647]: I think {: .note} should go under this sentence to render properly.

File: _ml-commons-plugin/api.md [hunk 6]
```diff
@@ -729,8 +732,147 @@ GET /_plugins/_ml/stats
...
+
+Some algorithms, such as [Localization]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms#localization), don't require trained models. You can run no-model-based algorithms using the `execute` API.
 
+```json
+POST _plugins/_ml/_execute/<algorithm_name>
+```
 
+### Example: Execute localization 
 
+The following example uses the localization algorithm to find subset-level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies.
```
Comment (cwillum) [inline _ml-commons-plugin/api.md:745]: "Localization"  for Localization algorithm should be capitalized here as well, I believe. 
```suggestion
The following example uses the Localization algorithm to find subset-level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies.
```

**Review by cwillum — APPROVED** [review]
LGTM. Just one suggestion for initial caps.

**Pull request merged**